src/components/fields/location-field/location-modal/types.ts

```diff
@@ -19,4 +19,5 @@ export interface ILocationModalProps
 	onClose: () => void;
 	onConfirm: (values: ILocationFieldValues) => void;
 	updateFormValues: (values: ILocationFieldValues, shouldDirty?: boolean) => void;
+	locationListTitle?: string;
```

Comment:
- optional keys should be a union with undefined to support projects using exactOptionalPropertyTypes in tsconfig

```suggestion
	locationListTitle?: string | undefined;
```